statesman-model/src/main/java/io/appform/statesman/model/action/ActionType.java

```diff
@@ -2,5 +2,7 @@
 
 public enum  ActionType {
     HTTP,
+    ROUTED_HTTP,
```

Comment:
Rename these to RUTED and COMPUND. The implementations do not assume the leaf action to be HTTP..